renderer/js/seq-template.js

```diff
@@ -0,0 +1,66 @@
+var SEQController = SEQController || {};
+SEQController.constant = {
+  qContainerElement: ".sequencing-content-container",
+  bgColors: ["#5DC4F5", "#FF7474", "#F9A817", "#48DCB6", "#5B6066"],
+  bgColor: "#5DC4F5"
+};
+
+/**
+ * intializes renderer template html controller and provides renderer plugin data with controller, 
+ * @param {Object} pluginInstance
+ * @memberof org.ekstep.questionunit.seq.seq-template
+ */
+SEQController.initTemplate = function (pluginInstance) {
+  SEQController.pluginInstance = pluginInstance;
+  SEQController.constant.bgColor = SEQController.constant.bgColors[_.random(0, SEQController.constant.bgColors.length - 1)];
+  SEQController.bgLeftCircleTop = _.random(-6, 6) * 10;
```

Comment:
`SEQController.bgLeftCircleTop = _.random(-6, 6) * 10` - is probably not used anywhere. Please check and remove